indexing-service/src/main/java/org/apache/druid/indexing/overlord/ForkingTaskRunner.java

```diff
@@ -714,6 +701,21 @@ private void saveRunningTasks()
     }
   }
 
+  private void shutdownTaskProcess(ForkingTaskRunnerWorkItem taskInfo)
```

Comment:
Would you add a comment like "this method calls lifecycle.stop()"?

Reply:
It sort of seems like a misleading comment, since it doesn't call that directly, it's just closing the stream of the forked task and destroying it if that encounters an exception. Also since I am not entirely sure that lifecycle stop _wasn't_ being called previously...

Reply:
That said, I agree it probably needs _some_ sort of javadoc/comment, i'll see what I can come up with

Reply:
added javadoc